React Implementation - Sprint 1

OliOliKT opened the pull request.

OliOliKT committed:
Add font styles and remove unnecessary .git files

emhberg committed:
IndividualPost done, and minor changes to footer and topbar

I created the IndividualPost page, and then I also created UserCommentsComponent on the same branch... upsi.. I created an initial array for the comments (can be changed later). Other than that I made some minor changes to footer and header, so they work optimal now.

emhberg committed:
Merge pull request #5 from OliOliKT/SingularPost

IndividualPost done, and minor changes to footer and topbar

BuurPrime committed:
Few additions

jdisitu committed:
Add UserSettingsSidebar component

OliOliKT committed:
Make PostStats, PostFilter components and make MyPosts page

jdisitu committed:
Update sidebar styling and structure

jdisitu committed:
Add new styling to navigation sidebar

OliOliKT committed:
Add User Settings to MyPosts

emhberg committed:
Changed small things to the UserCommentComponent.

Added a border to the UserCommentComponent, so it it will be possible to like a comment in the future.
Minor changes was also done to the topBar, as it acted a bit strange. So now it is always on top of the other components.

emhberg committed:
Merge pull request #6 from OliOliKT/SingularPost

Changed small things to the UserCommentComponent.

jdisitu committed:
add entire user settings page with all its components

jdisitu committed:
Add UserSettings page

mircealungu commented on src/components/CommentComponent.js:
```diff
@@ -13,7 +13,7 @@ const CommentBox = () => {
     <div>
       <textarea
-        id="comments"
+        id="comment"
```
Element id should be unique in the app. This looks very _un-unique_.

mircealungu commented on src/components/PostFilter.js:
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import './PostFilter.css';
```
Nice that you're importing the styles from the corresponding css file!

mircealungu commented on src/components/PostFilter.js:
```diff
@@ -0,0 +1,23 @@
+
+function PostFilter() {
+    return (
+        <div id="filter">
```
id should be unique.

jdisitu replied:
on it

OliOliKT committed:
Small modifications to a lot of components and pages

OliOliKT committed:
Merge pull request #8 from OliOliKT/modificationsForSprint1

Small modifications to a lot of components and pages

mircealungu commented on src/pages/IndividualPost.css:
```diff
@@ -0,0 +1,4 @@
+.IndividualPost {
```
https://github.com/itu-tid/lecture-notes/blob/main/Tech-TopUps/5-Coding-Practices.md#name-your-css-classes-with-kebab-case

mircealungu reviewed (changes requested):
overall good. a few requests for change

OliOliKT committed:
Refactor all files

OliOliKT committed:
Merge pull request #9 from OliOliKT/modificationsForSprint1

Refactor all files

OliOliKT committed:
Remove unnecessary files

OliOliKT committed:
Merge pull request #10 from OliOliKT/modificationsForSprint1

Remove unnecessary files

BuurPrime committed:
Added routing and dropdown menu in topbar component

- Changed topbarcomponent alot (both style and functionality)
- The Feed Navigation works, but does not change color accordingly
- Noticed that something is off on the pages in terms of spacing

emhberg committed:
Initial Log in page created

Initial Log in page created

BuurPrime committed:
Merge branch 'development' of https://github.com/OliOliKT/StealthStories into development

OliOliKT committed:
Change MyPosts so that it uses center instead of margin

OliOliKT committed:
Merge pull request #11 from OliOliKT/testBranchOliver

Change MyPosts so that it uses center instead of margin

BuurPrime committed:
Fixed footer and Trending block

emhberg committed:
Database for User created

Imported parse library and tested it out for user. Some navigation has also be changed, so it is up to date.

emhberg committed:
Merge pull request #12 from OliOliKT/TryingDataBaseForUser

Database for User created

OliOliKT committed:
Connect feed to database

OliOliKT committed:
Merge pull request #13 from OliOliKT/testBranchOliver

Connect feed to database

emhberg committed:
Creating a new user

Things added to create a new user, I also had to add a logout function, so changes have been done to the topbar as well.

emhberg committed:
Merge pull request #14 from OliOliKT/CreateUserPage

Creating a new user

jdisitu committed:
Added post functioanlity for liking and writing comments on posts

BuurPrime committed:
tried fixing the navigation

BuurPrime committed:
Merge branch 'discoverFeed' into development

OliOliKT committed:
Add post functionality via database

BuurPrime committed:
Fized trending box (again...) and fixed h1 margin issue

BuurPrime committed:
Fixed feedNavigation

OliOliKT committed:
Merge pull request #15 from OliOliKT/testBranchOliver

Add post functionality via database

emhberg committed:
Merge pull request #16 from OliOliKT/PostFunctionality

Added post functioanlity for liking and writing comments on posts

BuurPrime committed:
Small changes to individualpost

Commit:
add comment button redirection to individual page

emhberg committed:
created error messages for new user and for logging in

BuurPrime committed:
Merge pull request #17 from OliOliKT/CreateAlertForWrongPassword

created error messages for new user and for logging in

emhberg committed:
Merge pull request #18 from OliOliKT/PostFunctionality

add comment button redirection to individual page

Commit:
fixed bug where comment would lead to undefined URL. Add logo to Individual posts

Commit:
add functionality to enter button

emhberg committed:
The current user in user settings

The current user is now displayed in the user settings together with its ID. I deleted the "edit" button as well, as it was wrong that a user could change its email and ID

emhberg committed:
Merge pull request #19 from OliOliKT/GettingCurrentUserInUserSettings

The current user in user settings

Commit:
fixed merge conflicts

Commit:
fixed issue where shift enter also posted

jdisitu committed:
Merge pull request #20 from OliOliKT/PostFunctionality

Post functionality

OliOliKT committed:
Connect likes to database

OliOliKT committed:
Fix merge conflict

jdisitu committed:
Merge pull request #21 from OliOliKT/sprint2BranchOliver

Sprint2 branch oliver

emhberg committed:
Added gaps to posts

emhberg committed:
Merge pull request #22 from OliOliKT/FixingPosts

Added gaps to posts

emhberg commented:
Some of these changes, like variable names and kebab case will be changes for the next sprint.
